package.json

```diff
@@ -70,5 +75,60 @@
     "webpack-dev-server": "^2.8.1",
     "webpack-merge": "^4.1.0",
     "webpack-s3-plugin": "^1.0.0-rc.0"
+  },
+  "lint-staged": {
```

Comment:
eslint / prettier needed its config. I like to do this in the package.json now rather than separate files